scripts/env.list

```diff
@@ -11,6 +10,11 @@ PCS_IOTHUBREACT_HUB_PARTITIONS
 PCS_IOTHUBREACT_AZUREBLOB_ACCOUNT
 PCS_IOTHUBREACT_AZUREBLOB_KEY
 PCS_IOTHUBREACT_AZUREBLOB_ENDPOINT_SUFFIX
+PCS_ASA_DATA_AZUREBLOB_ACCOUNT
+PCS_ASA_DATA_AZUREBLOB_KEY
+PCS_ASA_DATA_AZUREBLOB_ENDPOINT_SUFFIX
+PCS_EVENTHUB_CONNSTRING
+PCS_EVENTHUB_NAME
```

Comment:
With the work being done on rules&actions, assuming it's going to use eventhub, this variable name is going to be an issue I guess. 

Also, what's the difference between `PCS_IOTHUBREACT_HUB_NAME` and `PCS_EVENTHUB_NAME` ?

Reply:
iothubreact_hub_name is the event hub compatible iot hub endpoint. EventHub_Name is the actual event hub used for device notifications. We will probably have to do a rework of env vars in the future, there is lots of naming confusion.